apis/python/node/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+requires = ["maturin>=0.12,<0.13"]
```

Comment:
This should be `maturin>=0.13,<0.14` since you are using `features` in `[tool.maturin]` instead of the removed `cargo-extra-args`.

Reply:
Agreed thanks! Copy pasted it from a previous crate, without paying attention

Reply:
Fixed in #60